code/modules/modular_computers/file_system/programs/security/implant_tracker.dm

```diff
@@ -0,0 +1,118 @@
+/datum/computer_file/program/implant_tracker
+	filename = "implnttrckr"
+	filedesc = "Remote Implant Tracking"
+	program_icon_state = "security"
+	extended_desc = "This program allows authorized security personnel to track and remotely interface with active implants."
+	required_access_run = access_armory
+	requires_ntnet = TRUE
+	available_on_ntnet = FALSE
+	size = 6
+	usage_flags = PROGRAM_CONSOLE
+	color = LIGHT_COLOR_ORANGE
+
+/datum/computer_file/program/implant_tracker/ui_interact(mob/user)
+	var/datum/vueui/ui = SSvueui.get_open_ui(user, src)
+	if(!ui)
+		ui = new /datum/vueui/modularcomputer(user, src, "mcomputer-security-implanttracker", 650, 500, "Remote Implant Tracking")
+		ui.auto_update_content = TRUE
+	ui.open()
+
+/datum/computer_file/program/implant_tracker/vueui_transfer(oldobj)
+	. = FALSE
+	var/uis = SSvueui.transfer_uis(oldobj, src, "mcomputer-security-implanttracker", 650, 500, "Remote Implant Tracking")
+	for(var/tui in uis)
+		var/datum/vueui/ui = tui
+		ui.auto_update_content = TRUE
+		. = TRUE
+
+/datum/computer_file/program/implant_tracker/vueui_data_change(var/list/data, var/mob/user, var/datum/vueui/ui)
+	. = ..()
+	data = . || data || list()
+
+	// Gather data for computer header
+	var/headerdata = get_header_data(data["_PC"])
+	if(headerdata)
+		data["_PC"] = headerdata
+		. = data
+
+	var/list/chem_implants = list()
+	for(var/obj/item/implant/chem/C in implants)
+		if(!C.implanted)
+			continue
+		var/turf/Tr = get_turf(C)
+		if(!Tr || !isStationLevel(Tr.z))
+			continue
+		var/list/chem_info = list(
+			"implanted_name" = C.imp_in.real_name,
+			"remaining_units" = round(C.reagents.total_volume, 0.1),
+			"ref" = "\ref[C]"
+			)
+		chem_implants[++chem_implants.len] = chem_info
+	VUEUI_SET_CHECK(data["chem_implants"], chem_implants, ., data)
```

Comment:
VUEUI_SET_CHECK does not work for lists. It will always update.

Either change it to just:
data["chem_implants"] = chem_implants

Or use VUEUI_SET_CHECK_LIST. 

We don't actually need to be using VUEUI_SET_CHECK in the vast majority of cases.